server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/QGateTester.java

```diff
@@ -57,6 +57,7 @@ void deleteAll() {
     }
     session.wsClient().qualitygates().list().getQualitygatesList().stream()
       .filter(qualityGate -> !qualityGate.getIsDefault())
+      .filter(qualityGate -> !qualityGate.getIsBuiltIn())
```

Comment:
Something is strange => I've set the built-in QG as default line 56, so the built-in and the default should be the same.

Reply:
Oh I understand ! It's because the setAsDefault is still using the setting...